app/src/controllers/project/sagas.js

```diff
@@ -485,6 +486,20 @@ function* watchUpdateProjectFilterPreferences() {
   yield takeEvery(UPDATE_PROJECT_FILTER_PREFERENCES, updateProjectFilterPreferences);
 }
 
+function* navigateToProject({ payload }) {
+  const { project } = payload;
+  const { organizationSlug, projectSlug } = project;
+
+  yield put({
+    type: PROJECT_PAGE,
+    payload: { organizationSlug, projectSlug },
+  });
+}
+
+function* watchNavigateToProject() {
+  yield takeEvery(NAVIGATE_TO_PROJECT, navigateToProject);
+}
```

Comment:
Do we have any places where this action is dispatched right now?

Reply:
Yes. This is project navigation.

Reply:
Done.